harbor/datadog_checks/harbor/harbor.py

```diff
@@ -108,18 +99,16 @@ def _submit_disk_metrics(self, api, base_tags):
 
     def check(self, instance):
         harbor_url = instance["url"]
-        username = instance["username"]
-        password = instance['password']
         tags = instance.get("tags", [])
         try:
             api = HarborAPI(harbor_url, self.http)
-            api.authenticate(username, password)
```

Comment:
So I assume the authentication will now be handled by the `RequestsWrapper` instance (passed above via `self.http`)  which automatically populates `auth` based on the config, correct?